commons/src/main/res/values-pl/strings.xml

```diff
@@ -821,51 +821,51 @@
-    <string name="translation_azerbaijani">Azerbaijani</string>
-    <string name="translation_breton">Breton</string>
-    <string name="translation_catalan">Catalan</string>
-    <string name="translation_czech">Czech</string>
-    <string name="translation_welsh">Welsh</string>
-    <string name="translation_danish">Danish</string>
-    <string name="translation_german">German</string>
-    <string name="translation_greek">Greek</string>
-    <string name="translation_spanish">Spanish</string>
-    <string name="translation_basque">Basque</string>
-    <string name="translation_persian">Persian</string>
-    <string name="translation_finnish">Finnish</string>
-    <string name="translation_french">French</string>
-    <string name="translation_galician">Galician</string>
-    <string name="translation_hindi">Hindi</string>
-    <string name="translation_croatian">Croatian</string>
-    <string name="translation_hungarian">Hungarian</string>
-    <string name="translation_indonesian">Indonesian</string>
-    <string name="translation_italian">Italian</string>
-    <string name="translation_hebrew">Hebrew</string>
-    <string name="translation_japanese">Japanese</string>
-    <string name="translation_korean">Korean</string>
-    <string name="translation_lithuanian">Lithuanian</string>
-    <string name="translation_norwegian">Norwegian</string>
-    <string name="translation_dutch">Dutch</string>
-    <string name="translation_polish">Polish</string>
-    <string name="translation_portuguese">Portuguese</string>
-    <string name="translation_russian">Russian</string>
-    <string name="translation_slovak">Slovak</string>
-    <string name="translation_slovenian">Slovenian</string>
-    <string name="translation_serbian">Serbian</string>
-    <string name="translation_swedish">Swedish</string>
-    <string name="translation_tamil">Tamil</string>
-    <string name="translation_turkish">Turkish</string>
-    <string name="translation_ukrainian">Ukrainian</string>
-    <string name="translation_vietnamese">Vietnamese</string>
-    <string name="translation_chinese">Chinese</string>
+    <string name="contributors">Współtwórcy</string>
+    <string name="translation">Tłumaczenie</string>
+    <string name="development">Rozwój</string>
+    <string name="other_help">Inna pomoc</string>
+    <string name="contributors_label">Ta lista zawiera wszystkich, którzy wyraźnie pomogli przy którejkolwiek z prostych aplikacji od Simple Mobile Tools, nie tylko przy tej. Tłumaczeniami zajmują się wolontariusze;
+        daj nam znać, jeśli też chcesz pomóc. Instrukcja jak tłumaczyć znajduje się na https://github.com/SimpleMobileTools/General-Discussion#how-can-i-suggest-an-edit-to-a-file lub po prostu skontaktuj się z nami pod adresem hello@simplemobiletools.com, jeśli masz jakieś pytania.
+        Dziękuję wszystkim współtwórcom i innym wspierającym!</string>
+    <string name="translation_arabic">arabski</string>
```

Comment:
These will be on a separate line with some names under them. Shouldnt they begin with a capital letter? They might be all-caps too, I will have to test how it looks like.